iis-wf/iis-wf-transformers/src/test/resources/eu/dnetlib/iis/wf/transformers/metadataextraction/checksum/preprocessing/sampledataproducer/data/output_document_content_url.json

```diff
@@ -1 +1 @@
-{"id": "12345", "url": "http://someurl1.com", "mimeType": "pdf", "contentChecksum": "12345", "contentSizeKB": 0}
\ No newline at end of file
+{"id": "12345", "url": "http://someurl2.com", "mimeType": "pdf", "contentChecksum": "12345", "contentSizeKB": 0}
```

Comment:
This worries me, because it means outcome is somehow indeterminate. Even though both links should point to the same content (due to the same checksum).